packages/portalnetwork/src/client/provider.ts

```diff
@@ -34,7 +35,19 @@ export class UltralightProvider {
   }
 
   static async create(opts: Partial<PortalNetworkOpts>): Promise<UltralightProvider> {
-    const portal = await PortalNetwork.create(opts)
+    const finalOpts = {
+      ...DEFAULT_OPTS,
+      ...opts,
+    }
+    
+    if (!finalOpts.bindAddress) {
+      throw this.prototype.createError(
+        ERROR_CODES.INVALID_PARAMS,
+        'bindAddress is required for portal network initialization'
+      )
+    }
+
+    const portal = await PortalNetwork.create(finalOpts)
```

Comment:
@ScottyPoi what do you think about adding these defaults to `PortalNetwork.create` as well?  I'm trying to think of a reason we wouldn't want the bootnodes list to be an optional default in the `create` method for the base client and not just the provider here.

Reply:
I think `PortalNetwork.create` should still default to an empty bootnodes list so the client doesn't automatically ping all of the bootnodes every time we run a test